Sources/MapboxMaps/Gestures/GestureHandlers/PinchGestureHandler.swift

```diff
@@ -1,9 +1,11 @@
 import UIKit
+@_implementationOnly import MapboxCommon_Private
```

Comment:
This import can be removed now

Reply:
This one stays, but the one from `GestureOptions` goes :)